### DISCUSSION TITLE
Capacitor targetSdkVersion and android

### DISCUSSION THREAD

**Discussion by Jmezas** [HaylLtd/cordova-plugin-background-geolocation#183] [General]
I have capacitor Ionic application and I need background geolocation for it. I really appreciate your work guys. But I have some problems with integrating the plugin in my app.

Does it work with capacitor 5?
What versions should I use to make it work?

    minSdkVersion = 22
    compileSdkVersion = 33
    targetSdkVersion = 33

**Comment by HarelM** [comment]
Feel free to copy the relevant configuration:
https://github.com/IsraelHikingMap/Site/tree/main/IsraelHiking.Web
I use capacitor 5 IIRC.
